tests/pulses/test_envelope.py

```diff
@@ -19,7 +19,7 @@
     [
         Rectangular(),
         Gaussian(rel_sigma=5),
-        GaussianSquare(rel_sigma=5, width=0.9),
+        GaussianSquare(risefall=5, sigma=0.9),
```

Comment:
Sigma should be greater than 1 in theory, right? Assuming that the instrument has a sampling rate of at least 1 Gsps.

Reply:
To be consistent I would be put as sigma an integer value for all subsequent tests.